source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -41,6 +41,60 @@ _For more on linting (including standard settings and config) see the [linting][
 * Lint checks using [PyFlakes][]
 
 
+### Dependencies
+
+It is important to have a good strategy for specifying your project's Python
+libraries. There are two desirable characteristics:
+
+1. Reproducibility
+
+    Pin your full dependencies or you'll get unpredictability between your dev
+    environment and other environments. You want a new starter to avoid small
+    hard-to-spot problems. And you want parity between what you test locally,
+    what is tested by CI, and what you deploy, or you risk new issues appearing
+    on a live server. Additionally these things can be hard to diagnose.
+
+    Traditionally a `requirements.txt` lists the app's immediate dependencies,
+    but that leaves sub-dependencies unpinned. It makes sense to use a tool to
+    generate these, but we've noticed [minor problems with using
+    piptools/pip-compile and pipenv/Pipfile][dm-deps-commit].
+
+    **Recommendation**: Put your top-level requirements in a
+    `requirements-app.txt` file and use [this Makefile
```

Comment:
If we do want to recommend using this Makefile is there a better way to share it than linking directly to a commit which changes it?

Reply:
I definitely think that a "GDS Way" template python project would be a nice thing to have...

Reply:
Sounds good, but I'd encourage this to be a separate PR for someone.